hail/python/requirements.txt

```diff
@@ -1,11 +1,13 @@
-numpy<2
-pandas>0.22,<0.24
+aiohttp
 bokeh>1.1,<1.3
-pyspark>=2.4,<2.4.2
-parsimonious<0.9
-ipykernel<5
 decorator<5
-requests>=2.21.0,<2.21.1
 gcsfs==0.2.1
 hurry.filesize==0.9
-scipy>1.2,<1.4
\ No newline at end of file
+ipykernel<5
```

Comment:
I believe you're missing jwt here. If I check out this branch, `make-install-wheel`, and then do from `hailtop import pipeline` I get a `ModuleNotFoundError: No module named 'jwt'`.